pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2397,6 +2397,35 @@ func getActualWildcardAndLocalHost(node *model.Proxy) (string, string) {
 	return WildcardIPv6Address, LocalhostIPv6Address
 }
 
+// getActualInboundBindAddress decide which ip should be bind for inbound.
+// default return 127.0.0.1/::1, if user specific use podIP in annotation,
+// return the podIP for binding
+func getActualInboundBindAddress(node *model.Proxy, port int) string {
+	bindPodIP := false
+	for _, p := range node.Metadata.BindPodIPPorts {
+		if p == port {
+			bindPodIP = true
+			break
+		}
+	}
+	for i := 0; i < len(node.IPAddresses); i++ {
```

Comment:
The first addr is podip, no need iterate here.

Reply:
I would like to keep the iteration at this version, this function is just like `getActualWildcardAndLocalHost`